com.arteeh.Flasher.yml

```diff
@@ -62,7 +62,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/arteeh/pinetime-flasher.git
-        tag: 0.1
-        commit: 0ea4fcec3bd6f88dbf635df724c7346aaa2dcd30
+        tag: 1.0
```

Comment:
The build subdirctory does not exist anymore https://github.com/arteeh/pinetime-flasher/tree/1.0/code and you removed the makefile in this tag

Let's not update the tag, it's fine to use the old one.

Reply:
I think I do have to move to a new commit, because when I added the EOL file the build failed due to missing categories and broken screenshot URLs. That's something I have to fix in .metainfo.xml in the pinetime-flasher repo.

Made a bit of a mess yesterday and then concluded that I should grab an older commit which last produced a working build, and from there update the metainfo.xml to fix the categories and screenshots, and then make a tag from that and refer to it here.

Or just remove the building of my program from the Yaml here and hope Flathub CI lets me build a flatpak with no app inside (which I suppose is harmless due to EOL)

I have time to fix this later today.

And sorry for being late on all the opened issues, Apparently I didn't have notifications on this repo.

Reply:
Yes feel free to patch the metainfo file, it's probably easier.